local/bin/py/build/actions/integrations.py

```diff
@@ -864,6 +866,22 @@ def process_integration_readme(self, file_name, marketplace=False, content=None)
             #         print(f"removing {integration_name} due to is_public/display_on_public_websites flag, {out_name}")
             #         remove(out_name)
 
+    def retrieve_page_description(self, page_url):
+        page = requests.get(page_url, timeout=1)
+        page_content = bs(page.content, 'html.parser')
+        metas = page_content.find_all('meta')
+        page_description = ''
```

Comment:
nit: It might be better to have this initialized to `None` since you're overriding it with `if not page_description:` . It functions the same, but I think it's clearer with None vs. an empty string